pom.xml

```diff
@@ -198,54 +198,5 @@
         </plugins>
       </build>
     </profile>
-    <profile>
-      <!-- Skip Sonar plugin execution if SONARCLOUD_TOKEN is not set. -->
-      <id>skipSonar</id>
-      <activation>
-        <property>
-          <name>!env.SONARCLOUD_TOKEN</name>
-        </property>
-      </activation>
-      <properties>
-        <sonar.skip>true</sonar.skip>
-      </properties>
-    </profile>
-    <profile>
-      <id>pullRequestAnalysis</id>
-      <!-- Only set sonar.pullrequest properties when the Travis build is triggered by a PR.
-           When the build is triggered by push to a branch or by cron, the consumed env.TRAVIS properties are empty,
-           resulting in invalid sonar.pullrequest properties and the Sonar plugin would fail. -->
-      <activation>
-        <property>
-          <name>env.TRAVIS_EVENT_TYPE</name>
-          <value>pull_request</value>
-        </property>
-      </activation>
-      <!-- See https://docs.travis-ci.com/user/environment-variables/ for list of Travis environment variables. -->
-      <!-- See https://docs.sonarqube.org/latest/analysis/pull-request/ for properties required by Sonar for PR analysis. -->
-      <properties>
-        <!--suppress UnresolvedMavenProperty -->
-        <sonar.pullrequest.branch>${env.TRAVIS_PULL_REQUEST_BRANCH}</sonar.pullrequest.branch>
```

Comment:
How does GitHub Actions CI provide these properties? Without these, the PR won't get paired with SonarCloud.

Reply:
Somehow Sonar can read them from the environment, see the log - it has all the info.